src/Sql/Sql.Test/ScenarioTests/DatabaseBackupTests.cs

```diff
@@ -102,6 +115,14 @@ public void TestLongTermRetentionV2()
                 RunPowerShellTest("Test-LongTermRetentionV2");
             }
         }
+
+        [Fact(Skip = "This is not recordable test")]
```

Comment:
Please do not add a skipped test.

Reply:
I have the test pass and recorded. However the test set up (i.e. ensure a backup exist) need at least 18 hours. That is why I marked it as skipped.

Reply:
so, if the test takes 18 hours, you should not check it in.  Please remove it.